classes/genre.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
```

Comment:
id should be moved to arre_accessor because we want to to set it.

Reply:
Thank you